foyer/forcefield.py

```diff
@@ -512,6 +512,11 @@ def apply(self, topology, references_file=None, use_residue_map=True,
             atom_types = set(atom.type for atom in structure.atoms)
             self._write_references_to_file(atom_types, references_file)
 
+        # TODO: Check against the name of the force field and/or store
+        # combining rule directly in XML, i.e.
+        # if self.name == 'oplsaa':
+        structure.combining_rule = 'geometric'
```

Comment:
Could we make this an optional arg passed to `apply()` that defaults to `'geometric'`?